engine-tests/src/tests/native_erc20_connector.rs

```diff
@@ -0,0 +1,441 @@
+// Simulation tests for exit to NEAR precompile.
+// Note: `AuroraRunner` is not suitable for these tests because
+// it does not execute promises; but `near-sdk-sim` does.
+mod sim_tests {
```

Comment:
Would be cool to have at least one test that runs through the standalone engine as well.